src/app/public/modules/radio/radio-group.component.spec.ts

```diff
@@ -17,29 +17,34 @@ import {
 import {
   SkyRadioFixturesModule,
   SkyRadioGroupBooleanTestComponent,
-  SkyRadioGroupTestComponent
+  SkyRadioGroupReactiveFixtureComponent,
+  SkyRadioGroupFixtureComponent
 } from './fixtures';
 
-describe('Radio group component', function () {
-  let fixture: ComponentFixture<SkyRadioGroupTestComponent>;
-  let componentInstance: SkyRadioGroupTestComponent;
+//#region helpers
+function getRadios(radioFixture: ComponentFixture<any>): NodeListOf<HTMLInputElement> {
+  return radioFixture.nativeElement.querySelectorAll('.sky-radio-input');
+}
 
-  //#region helpers
-  function getRadios(radioFixture: ComponentFixture<any>) {
```

Comment:
A lot of our helpers weren't typecast and not available in a global context. I moved these outside the `describe` block so I could use them between different fixtures.